Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>mini batches is a very ML kind of jargon that I'm not sure applies here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>yeah, iterations is a better word</p>